ros_ws/src/car_tf/src/laserscan_transformer.cpp

```diff
@@ -0,0 +1,22 @@
+#include "laserscan_transformer.h"
+#include <laser_geometry/laser_geometry.h>
+#include <ros/ros.h>
+#include <tf/transform_listener.h>
+
+constexpr const char* TOPIC_LASER_SCAN = "/racer/laser/scan";
+constexpr const char* TOPIC_LASER_SCAN_POINTCLOUD = "/racer/laser/tf_pointcloud";
+constexpr const char* MODEL_BASE_LINK = "base_link";
```

Comment:
These constants are usually in the `.h` file, so you could move them for consistency.